### PR TITLE
fix: configmap-pod CrashLoopBackOff

### PR DESCRIPTION
**Pull request by hchyue**
with the "configmap-pod" yaml,the container "test"fail to run with reason of "CrashLoopBackOff".  
I suggest adding  a command.

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -174,6 +174,7 @@ spec:
   containers:
     - name: test
       image: busybox:1.28
+      command: ['sh', '-c', 'echo The app is running! && sleep 3600']
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md:177]: How about:
```suggestion
      command: ['sh', '-c', 'echo "The app is running!" && tail -f /dev/null']
```

Comment (hchyue) [reply content/en/docs/concepts/storage/volumes.md:177]: yes，it is much better.

**Review by hchyue — COMMENTED** [review]
"tail -f /dev/null" is better than "sleep 3600"

**Comment by hchyue** [comment]
@sftim  Is there anything else that needs to be done from my end?

**Pull request merged**